test-case/check-pause-release-suspend-resume.sh

```diff
@@ -51,47 +51,50 @@ set -e
 ##  * No unexpected errors should be present in dmesg during or after test
 ##      completion.
 
+# shellcheck source=case-lib/lib.sh
 source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
```

Comment:
No, not completely. Can you please run shellcheck locally before pushing? Especially after fixing shellcheck issues? It would have saved a lot of time.
```shell
sudo apt install shellcheck
shellcheck -x test-case/check-pause-release-suspend-resume.sh
```